Directory.Packages.props

```diff
@@ -27,8 +27,9 @@
     <PackageVersion Include="Nswag.MSbuild" Version="14.0.7" />
     <PackageVersion Include="ReportGenerator" Version="5.2.5" />
     <PackageVersion Include="System.Text.Json" Version="4.6.0" />
+    <PackageVersion Include="Verify.Xunit" Version="24.2.0" />
```

Comment:
Could you add this to the dependabot group for xunit please?

https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/ace3015e98631f02c05f66ff294158dcdf2ccaf2/.github/dependabot.yml#L33-L35